<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Automatic Shades</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(100, 100, 100);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_10">
    <item>
     <widget class="QFrame" name="frame_top">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="button_home">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	background-color: rgb(143, 143, 143);
	border: none;
	border-radius: 10px;
	padding: 10px;
	margin-bottom: 10px;
}

QPushButton:hover {
	background-color: rgb(154, 154, 154);
}

QPushButton:pressed {
	background-color: rgb(125, 125, 125);
}</string>
         </property>
         <property name="text">
          <string>Home</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_control">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	background-color: rgb(143, 143, 143);
	border: none;
	border-radius: 10px;
	padding: 10px;
	margin-bottom: 10px;
}

QPushButton:hover {
	background-color: rgb(154, 154, 154);
}

QPushButton:pressed {
	background-color: rgb(125, 125, 125);
}</string>
         </property>
         <property name="text">
          <string>Control</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_settings">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	background-color: rgb(143, 143, 143);
	border: none;
	border-radius: 10px;
	padding: 10px;
	margin-bottom: 10px;
}

QPushButton:hover {
	background-color: rgb(154, 154, 154);
}

QPushButton:pressed {
	background-color: rgb(125, 125, 125);
}</string>
         </property>
         <property name="text">
          <string>Settings</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_main">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(125, 125, 125);</string>
         </property>
         <property name="currentIndex">
          <number>2</number>
         </property>
         <widget class="QWidget" name="page_1_home">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QFrame" name="frame_home">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QFrame" name="frame_home_temperature">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_8">
                 <item alignment="Qt::AlignHCenter">
                  <widget class="QLabel" name="label_home_temperature">
                   <property name="font">
                    <font>
                     <pointsize>20</pointsize>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">color: rgb(255, 255, 255);</string>
                   </property>
                   <property name="text">
                    <string>Temperature: 0</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item alignment="Qt::AlignHCenter">
               <widget class="QLabel" name="label_home_status">
                <property name="font">
                 <font>
                  <pointsize>20</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 255, 255);</string>
                </property>
                <property name="text">
                 <string>Status:</string>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignBottom">
               <widget class="QLabel" name="version">
                <property name="font">
                 <font>
                  <pointsize>20</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 255, 255);</string>
                </property>
                <property name="text">
                 <string>v.0.0.1</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_2_control">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QFrame" name="frame_control">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QFrame" name="frame_control_top">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_6">
                 <item alignment="Qt::AlignHCenter">
                  <widget class="QLabel" name="label_control_status">
                   <property name="font">
                    <font>
                     <pointsize>20</pointsize>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">color: rgb(255, 255, 255);</string>
                   </property>
                   <property name="text">
                    <string>Status:</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QFrame" name="frame_control_middle">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_7">
                 <item>
                  <widget class="QFrame" name="frame_control_middle_left">
                   <property name="frameShape">
                    <enum>QFrame::StyledPanel</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_3">
                    <item>
                     <widget class="QPushButton" name="pushButton_control_open">
                      <property name="minimumSize">
                       <size>
                        <width>150</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>16777215</width>
                        <height>200</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <family>Segoe UI</family>
                        <pointsize>20</pointsize>
                       </font>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	background-color: rgb(143, 143, 143);
	border: none;
	border-radius: 10px;
	padding: 10px;
	margin-bottom: 10px;
	
}

QPushButton:hover {
	background-color: rgb(154, 154, 154);
}

QPushButton:pressed {
	background-color: rgb(125, 125, 125);
}</string>
                      </property>
                      <property name="text">
                       <string>Open</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="pushButton_control_close">
                      <property name="minimumSize">
                       <size>
                        <width>150</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>16777215</width>
                        <height>200</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <pointsize>20</pointsize>
                       </font>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	background-color: rgb(143, 143, 143);
	border: none;
	border-radius: 10px;
	padding: 10px;
	margin-bottom: 10px;
}

QPushButton:hover {
	background-color: rgb(154, 154, 154);
}

QPushButton:pressed {
	background-color: rgb(125, 125, 125);
}</string>
                      </property>
                      <property name="text">
                       <string>Close</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QFrame" name="frame_control_middle_right">
                   <property name="frameShape">
                    <enum>QFrame::StyledPanel</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_6">
                    <item>
                     <widget class="QPushButton" name="pushButton_control_up">
                      <property name="minimumSize">
                       <size>
                        <width>150</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>16777215</width>
                        <height>200</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <family>Segoe UI Emoji</family>
                        <pointsize>32</pointsize>
                        <weight>75</weight>
                        <bold>true</bold>
                       </font>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	background-color: rgb(143, 143, 143);
	border: none;
	border-radius: 10px;
	padding: 10px;
	margin-bottom: 10px;
}

QPushButton:hover {
	background-color: rgb(154, 154, 154);
}

QPushButton:pressed {
	background-color: rgb(125, 125, 125);
}</string>
                      </property>
                      <property name="text">
                       <string>↑</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="pushButton_control_down">
                      <property name="minimumSize">
                       <size>
                        <width>150</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>16777215</width>
                        <height>200</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <family>Segoe UI Emoji</family>
                        <pointsize>32</pointsize>
                        <weight>75</weight>
                        <bold>true</bold>
                       </font>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	background-color: rgb(143, 143, 143);
	border: none;
	border-radius: 10px;
	padding: 10px;
	margin-bottom: 10px;
}

QPushButton:hover {
	background-color: rgb(154, 154, 154);
}

QPushButton:pressed {
	background-color: rgb(125, 125, 125);
}</string>
                      </property>
                      <property name="text">
                       <string>↓</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QFrame" name="frame_control_bottom">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_9">
                 <item>
                  <widget class="QCheckBox" name="auto_man_switch">
                   <property name="styleSheet">
                    <string notr="true">QCheckBox::indicator{
	width: 50px;
	height: 50px;
}
QCheckBox::indicator:checked{
	image: url(:/images/images/on-button.png);
}

QCheckBox::indicator:unchecked{
	image: url(:/images/images/off-button.png);
}
</string>
                   </property>
                   <property name="text">
                    <string>Manual Controls</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_3_settings">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QFrame" name="frame_settings">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <widget class="QFrame" name="frame_settings_temperature">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <spacer name="horizontalSpacer_settings_temperature_1">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_settings_temperature">
                   <property name="minimumSize">
                    <size>
                     <width>200</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>200</width>
                     <height>142</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <pointsize>20</pointsize>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">color: rgb(255, 255, 255);</string>
                   </property>
                   <property name="text">
                    <string>Temperature: </string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSlider" name="horizontalSlider_settings_temperature">
                   <property name="minimumSize">
                    <size>
                     <width>300</width>
                     <height>50</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>450</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">QSlider::groove:horizontal {
border: 1px solid #bbb;
background: white;
height: 25px;
border-radius: 5px;
}

QSlider::sub-page:horizontal {
background: qlineargradient(x1: 0, y1: 0,    x2: 0, y2: 1,
    stop: 0 #66e, stop: 1 #bbf);
background: qlineargradient(x1: 0, y1: 0.2, x2: 1, y2: 1,
    stop: 0 #bbf, stop: 1 #55f);
border: 1px solid #777;
height: 25px;
border-radius: 5px;
}

QSlider::add-page:horizontal {
background: #fff;
border: 1px solid #777;
height: 20px;
border-radius: 5px;
}

QSlider::handle:horizontal {
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
    stop:0 #eee, stop:1 #ccc);
border: 1px solid #777;
width: 13px;
margin-top: -2px;
margin-bottom: -2px;
border-radius: 5px;
}

QSlider::handle:horizontal:hover {
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
    stop:0 #fff, stop:1 #ddd);
border: 1px solid #444;
border-radius: 5px;
}

QSlider::sub-page:horizontal:disabled {
background: #bbb;
border-color: #999;
}

QSlider::add-page:horizontal:disabled {
background: #eee;
border-color: #999;
}

QSlider::handle:horizontal:disabled {
background: #eee;
border: 1px solid #aaa;
border-radius: 5px;
}</string>
                   </property>
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_horizontalSpacer_settings_temperature_2">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Automatic_Blinds/resources.qrc"/>
 </resources>
 <connections/>
</ui>
